### ISSUE TITLE
comptability with PHP 8

### ISSUE THREAD

**Issue by vaishaliwebkul** [uvdesk/community-skeleton#358]
# Bug report

`php version - 8.0`

`composer version- 2.0`

![Screenshot_240](https://user-images.githubusercontent.com/43210406/106560056-61f07300-654c-11eb-9d95-2a8c5d28440d.png)

**Comment by papnoisanjeev** [comment]
@vaishaliwebkul 

As currently **doctrine/doctrine-migrations-bundle** [Used here](https://github.com/uvdesk/community-skeleton/blob/master/composer.json#L12) does not support **php 8**.

Check this: https://github.com/symfony/symfony/issues/39582#issuecomment-748639659

So may be we need to wait for it.

**Comment by PeopleInside** [comment]
Interesting Issue.

There is open issue about the components that doesn't support PHP 8?
The issue mentioned in the previous message is already closed so you will be not able to track progression or resolution of the issue.

For now PHP 7.4 seems will be supported for a year than will be obsolete... so maybe there are some time :)

**Comment by rabol** [comment]
Any news on this ?

**Comment by papnoisanjeev** [comment]
@rabol 

Yes, soon within few days will try to make a update on project for the same.

**Comment by Draecal** [comment]
Any update on compatibility? 
![image](https://user-images.githubusercontent.com/8697000/201998501-f605aca5-0c86-4a0c-9360-9aec982a0611.png)

**Comment by PeopleInside** [comment]
@Draecal I use UVdesk with PHP 8.1 but my setup is from the GitHub repo.
Maybe the repo discussed on this discussion is not supporting PHP 8.0 but I use UVdesk on PHP 8.1
https://cdn.uvdesk.com/uvdesk/downloads/opensource/uvdesk-community-current-stable.zip

**Comment by Draecal** [comment]
> @Draecal I use UVdesk with PHP 8.1 but my setup is from the GitHub repo.
> Maybe the repo discussed on this discussion is not supporting PHP 8.0 but I use UVdesk on PHP 8.1
> https://cdn.uvdesk.com/uvdesk/downloads/opensource/uvdesk-community-current-stable.zip

Managed to install but now I got this view![image_20221115211018.png](https://user-images.githubusercontent.com/8697000/202026338-a42d7178-6f69-4462-bf8a-5a9a5333e510.png)

**Comment by PeopleInside** [comment]
@Draecal maybe this can be a configuration issue on the server.
What is the web address loaded for see this page?
This include the public folder? https://support.domain.ext/uvdeskroot/public?

Have you checked the browser developer console?
If is a web install you can try to share the URL on live chat on my [helpdesk page](https://helpdesk.peopleinside.it/it/blog/free-limited-support-on-third-part-software) (Live chat will be online just for 30 min from now)

If is a local install and you never be able to figure how to resolve you need wait a UVdesk staff reply :)

**Comment by PeopleInside** [comment]
@Draecal thank you for your chat!
Seems you never uploaded all files so I suggested to follow what I linked in my [previous reply](https://github.com/uvdesk/community-skeleton/issues/358#issuecomment-1315803757) and to install UVdesk using FTP.

I hope this will resolve your image or files loading issue that generate 404 in the developer browser console.

**Comment by Draecal** [comment]
Completely fixed!!! thanks @PeopleInside
